packages/config-transformer/src/Command/SwitchFormatCommand.php

```diff
@@ -65,17 +67,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return self::SUCCESS;
     }
 
-    /**
-     * @return SmartFileInfo[]
-     */
-    private function findFileInfos(Configuration $configuration): array
-    {
-        $suffixes = $configuration->getInputSuffixes();
-        $suffixesRegex = '#\.' . implode('|', $suffixes) . '$#';
```

Comment:
The bug was here, the regex should have been wrapped in ()

```diff
- '#\.' . implode('|', $suffixes) . '$#';
+ '#\(.' . implode('|', $suffixes) . ')$#';
```

Otherwise it matched different files. The static version with regex101 link will be better :+1: